WordPress/Classes/ViewRelated/Comments/CommentModerationBar.swift

```diff
@@ -0,0 +1,132 @@
+import UIKit
+
+class CommentModerationBar: UIView {
+
+    // MARK: - Properties
+
+    @IBOutlet private weak var contentView: UIView!
+    @IBOutlet private weak var pendingButton: UIButton!
+
+    // MARK: - Init
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+
+        guard let view = loadViewFromNib() else {
+            DDLogError("CommentModerationBar: Failed loading view from nib.")
+            return
+        }
+
+        view.frame = self.bounds
+        configureView()
+        self.addSubview(view)
+    }
+
+}
+
+// MARK: - Private Extension
+
+private extension CommentModerationBar {
+
+    // MARK: - Configure
+
+    func loadViewFromNib() -> UIView? {
```

Comment:
Minor comment: what do you think about making this class conform to `NibLoadable` instead? Then we can remove this function and use the `loadFromNib()` static method instead.

Reply:
Oh I tried that. 😄 

This is a bit different situation. Because `CommentContentTableViewCell` is rendering `CommentModerationBar` from a xib, the setting are different.
- `CommentModerationBar`'s File Owner has to be set to the class (it is usually not) so `CommentContentTableViewCell` can find it.

<img width="1165" alt="Screen Shot 2021-09-24 at 11 27 07 AM" src="https://user-images.githubusercontent.com/1816888/134716563-bb2317d2-999f-42b9-b620-fee2bba3c2e6.png">

- Because of ☝️ , the View's class cannot be set (it usually is). Doing so causes `init` to be called infinitely because Class is already set above.

<img width="1163" alt="Screen Shot 2021-09-24 at 11 27 20 AM" src="https://user-images.githubusercontent.com/1816888/134716705-615f2974-e767-4b21-bd64-40eabf6070bf.png">


And this is where `NibLoadable` becomes unusable. It expects the View's class to match. In this case it does not, so `NibLoadable` crashes.

`let result = defaultBundle.loadNibNamed(defaultNibName, owner: nil, options: nil)?.first as? Self`

Reply:
Ah, got it. Thanks for explaining! 👍